src/components/GameView/ScrapDialog.vue

```diff
@@ -1,47 +1,55 @@
 <template>
-  <v-dialog v-model="show" max-width="650" scrollable>
+  <base-dialog v-model="show" id="scrap-dialog">
     <template #activator="{ props }">
       <span v-bind="props">
         <slot name="activator" />
       </span>
     </template>
-    <v-card id="scrap-dialog">
-      <v-card-title class="d-flex justify-space-between">
+    <template #title>
+      <div class="d-flex justify-space-between align-center w-100">
         <h1>Scrap Pile</h1>
-        <v-btn icon data-cy="close-scrap-dialog-x" @click="show = false">
-          <v-icon icon="mdi-close" size="large" />
-        </v-btn>
-      </v-card-title>
-      <v-card-text>
-        <div class="mt-4">
-          <card-list-sortable
-            :cards="scrap"
-            empty-text="There are no cards in the scrap pile."
-            data-selector-prefix="scrap-dialog"
-          />
-        </div>
-      </v-card-text>
-      <v-card-actions class="d-flex justify-end my-2">
-        <v-btn
-          color="primary"
-          variant="outlined"
-          data-cy="close-scrap-dialog-button"
+        <v-btn 
+          icon
```

Comment:
While you are at it, per #446, let's fix the button style to remove the background circle
```suggestion
          icon
          variant="text"
```